### PR TITLE
Adding some inventory examples

### PR DESCRIPTION
**Pull request by ericsjogren**

### CODE REVIEW

File: inventory.adoc [hunk 1]
```diff
@@ -4,7 +4,47 @@
...
+#### Example 1:
+`PUT /organizations/{organizationUuid}/inventory`
+
+I have 30 in stock and want to set to 40, then we just swap the locations.
+ ```json
+  "productUuid": <uuid>
+  "variantUuid": <uuid>,
+  "fromLocationUuid": <SUPPLIER_location_uuid>,
+  "toLocationUuid": “<STORE_location_uuid>,
+  "change": 10
```
Comment (adamvoncorswant) [inline inventory.adoc]: Wouldn't a better example be supplier->store, or store->bin? The scenario above (hopefully) doesn't happen very often and might be misleading?

Comment (adamvoncorswant) [reply inventory.adoc]: Never mind. I see that you already changed (or I misread before). :D

File: inventory.adoc [hunk 2]
```diff
@@ -4,7 +4,47 @@
 Production:: https://inventory.izettle.com
 
 ### Description
-The Inventory Service keeps track of the stock level of products. In the API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
+The Inventory Service keeps track of the stock level for variants within products. The API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
```
Comment (adamvoncorswant) [inline inventory.adoc]: perhaps clarify: "The service automatically decreases the stock when purchases are made with the iZettles POS application."

File: inventory.adoc (hunk 1 above)
Comment (ericsjogren) [reply inventory.adoc]: You probably read the first example, where i sent store -> supplier, which is a bad example, fixed that :) thanks!

**Comment by ericsjogren** [comment]
@psvanstrom Do you feel like anything is missing?

File: inventory.adoc [hunk 3]
```diff
@@ -4,7 +4,47 @@
 Production:: https://inventory.izettle.com
 
 ### Description
-The Inventory Service keeps track of the stock level of products. In the API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
+The Inventory Service keeps track of the stock level for variants within products. The API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is made within the iZettle POS application. Currently the API is only available in the production environment requiring production credentials to access.
```
Comment (guillermo-carrasco) [inline inventory.adoc]: > [requiring production credentials to access](https://www.izettle.com/api-access) 

maybe? :slightly_smiling_face: take it or leave it

File: inventory.adoc [hunk 4]
```diff
@@ -4,7 +4,47 @@
...
+
+For inventory we use a concept of transactions, where we always move stock between so called `Locations`.
+Locations are created automatically when tracking is enabled on any product. the locations that currently exists and the natural flow of stock balance is shown below:
```
Comment (guillermo-carrasco) [inline inventory.adoc]: capital T after the dot :nerd_face:

**Pull request merged**